vendor/modules.txt

```diff
@@ -11,10 +11,12 @@ github.com/PuerkitoBio/purell
 # github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578
 github.com/PuerkitoBio/urlesc
 # github.com/ahmetb/gen-crd-api-reference-docs v0.1.5
+## explicit
```

Comment:
These changes are related to 1.14?

Reply:
Correct, it's changed after `make revendor`.

Reply:
Thanks!